packages/frontend/src/pages/Flows/index.tsx

```diff
@@ -164,16 +129,9 @@ export default function Flows(): React.ReactElement {
             sx={{ display: 'flex', justifyContent: 'center', mt: 3 }}
             page={pageInfo?.currentPage}
             count={pageInfo?.totalPages}
-            onChange={(event, page) =>
+            onChange={(_event, page) =>
               setSearchParams({ page: page.toString() })
             }
```

Comment:
I realised we need to add a check here if a user clicks on page 1:
`setSearchParams(page === 1 ? {} : { page: page.toString() })`

Reply:
Makes sense, gogogogo!